recipes/tardigrade_error_tools/meta.yaml

```
@@ -0,0 +1,95 @@
+{% set name = "tardigrade_error_tools" %}
+{% set version = "0.9.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/UCBoulder/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: af55feef37f99979c3ccfb20c2f563b5e10175cb7599264b441c9608fdb68652
+
+build:
+  number: 0
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  script:
+    - set -x
+    # Build and install c++ libraries
+    - cmake -S . -B build -DCMAKE_BUILD_TYPE=Release ${CMAKE_ARGS}
+    - cmake --build build --target tardigrade_error_tools tardigrade_error_tools_PYTHON
+    - cmake --install build --prefix ${PREFIX}
+    # Build and install Conda package
+    - $PYTHON -m pip install build/src/python/dist/$PKG_NAME-$PKG_VERSION.tar.gz --no-deps --ignore-installed -v --no-build-isolation
+    - $PYTHON -m pip show $PKG_NAME
+  missing_dso_whitelist:
+      - /lib64/librt.so.1
+      - /lib64/libc.so.6
+      - /lib64/libpthread.so.0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - cython
+    - python
+    - breathe >=4.3.0
+    - sphinx >=3.0.4
+    - sphinx_rtd_theme >=0.4.3
+    - sphinxcontrib-bibtex
+    - setuptools_scm[toml] >=6.2
+  host:
+    - {{ compiler('cxx') }}
+    - python {{ python }}
+    - python-build
+    - pip
+    - cython
+    - setuptools >=62
+    - setuptools_scm >=6.2
+    - make
+    - cmake >=3.18
+    - eigen >=3.3.7
+    - boost >=1.59
+    - doxygen
+  run:
+    - python
+    - eigen >=3.3.7
```

```suggestion
requirements:
  build:
    - {{ compiler('c') }}
    - {{ stdlib('c') }}
    - {{ compiler('cxx') }}
    - make
    - cmake >=3.18
  host:
    - python {{ python }}
    - breathe >=4.3.0
    - sphinx >=3.0.4
    - sphinx_rtd_theme >=0.4.3
    - sphinxcontrib-bibtex
    - python-build
    - pip
    - cython
    - setuptools >=62
    - setuptools_scm >=6.2
    - eigen >=3.3.7
    - boost >=1.59
    - doxygen
  run:
    - python
```
I've cleaned up and rearranged the dependencies here to better match what we expect for native packages.  `build` dependencies must match the architecture of whatever machine we are building on, while `host` packages must match the target architecture.  Some more work will probably be needed when cross-compilation is enabled in the feedstock but this should get us closer to what we want.